issue90_test.go

```diff
@@ -0,0 +1,56 @@
+package mergo
+
+import (
+	"testing"
+
+	"github.com/magiconair/properties/assert"
+)
+
+func TestMergoSimpleMap(t *testing.T) {
```

Comment:
I think we already have a test like that somewhere, no need to duplicate (maybe I'm wrong)